src/pages/Bridge/index.tsx

```diff
@@ -298,135 +299,138 @@ export default function Bridge() {
-                showWalletConnector={false}
-                parentRef={toPanelRef}
-                show={showToList}
-                onOuterClick={() => setShowToList(false)}
-                placement="bottom"
-              />
-            </AssetWrapper>
-          </Row>
-          <CurrencyInputPanelBridge
-            value={isCollecting && collectableTx ? collectableTx.fromValue : typedValue}
-            displayedValue={displayedValue}
-            setDisplayedValue={setDisplayedValue}
-            currency={isCollecting ? collectableCurrency : bridgeCurrency}
-            onUserInput={onUserInput}
-            onMax={isCollecting ? undefined : handleMaxInput}
-            onCurrencySelect={onCurrencySelection}
-            disableCurrencySelect={!account || isCollecting || !isNetworkConnected || !hasBridges}
-            disabled={!account || isCollecting || !isNetworkConnected}
-            id="bridge-currency-input"
-            hideBalance={
-              isCollecting && collectableTx
-                ? ![collectableTx.fromChainId, collectableTx.toChainId].includes(chainId ?? 0)
-                : false
-            }
-            isLoading={!!account && isNetworkConnected && listsLoading}
-            chainIdOverride={isCollecting && collectableTx ? collectableTx.toChainId : undefined}
-            maxAmount={maxAmountInput}
-            isOutputPanel={false}
-          />
-          {activeTab === BridgeTab.BRIDGE_SWAP && (
-            <OutputPanelContainer>
-              <CurrencyInputPanelBridge
-                id="bridge-currency-output"
-                value={toValue}
-                onUserInput={onUserInput}
-                disabled={true}
-                currency={bridgeOutputCurrency}
-                onCurrencySelect={onCurrencyOutputSelection}
-                isOutputPanel={true}
-                disableCurrencySelect={!account || isCollecting || !isNetworkConnected || !hasBridges}
-                isLoading={!!account && isNetworkConnected && listsLoading}
-              />
-            </OutputPanelContainer>
-          )}
-
-          <BridgeActionPanel
-            account={account}
-            fromNetworkChainId={fromChainId}
-            toNetworkChainId={isCollecting && collectableTx ? collectableTx.toChainId : toChainId}
-            handleModal={handleModal}
-            handleCollect={handleCollect}
-            isNetworkConnected={isNetworkConnected}
-            isCollecting={isCollecting}
-            setIsCollecting={setIsCollecting}
-          />
-        </AppBody>
-      )}
-      {(activeTab === BridgeTab.BRIDGE || activeTab === BridgeTab.BRIDGE_SWAP) && showAvailableBridges && (
-        <BridgeSelectionWindow />
-      )}
-      {!!bridgeSummaries.length && (
-        <BridgeTransactionsSummary
-          extraMargin={activeTab !== BridgeTab.HISTORY && !showAvailableBridges}
-          transactions={bridgeSummaries}
+    <>
+      <PageMetaData title="Bridge | Swapr" />
```

Comment:
Same, we might want to translate titles

Reply:
We need to change `og:title` `og:description` `twitter:title` `twitter:description` also get the full benifit from social media as well. Will do translations along with this change. Since this fixes fathom analytics as well. I will merge this and create a new issue for addition translations + social media .